packages/ui/src/common/model/breadcrumbs.ts

```diff
@@ -0,0 +1,11 @@
+export const knownRoutes = [
+  { path: '/profile', breadcrumb: 'My Profile' },
+  { path: '/profile/memberships', breadcrumb: 'My Memberships' },
+  { path: '/working-groups', breadcrumb: 'Working Groups' },
+  { path: '/working-groups/openings', breadcrumb: 'Openings' },
+  { path: '/working-groups/my-applications', breadcrumb: 'My Applications' },
+  { path: '/working-groups/my-roles', breadcrumb: 'My Roles' },
+  { path: '/members', breadcrumb: 'Members' },
+]
+
+export const excludePaths = ['/', '/working-groups/grouppreview']
```

Comment:
Not sure if this belongs in `model`, couldn't find a better place for it though

Reply:
I don't have a better idea for the second level, but the first level (`common`) is IMO wrong.

The know paths are tied to the application, so they should be moved to the `src/app`.

This will make you change how the `<Breadcrumbs />` component is created in terms of inversion of control. The known paths should be injected as a dependency (prop) to the component.

When working on that please check if some memo isn't needed somewhere to prevent re-renders of the breadcrumbs when the page is changed (re-rendered) but the URL stays the same.